pages/news_item/news_item.ejs

```diff
@@ -6,31 +6,38 @@
   </head>
   <body>
     <%- include('../partials/navbar', {navPage: 'news_item'}); %>
-    <div id="content" class="container">
+    <article id="content" class="container">
 
-      <a class="btn btn-primary mb-4" href="<%= urlPrefix %>/news_items" role="button">
-        <i class="fa fa-arrow-left" aria-hidden="true"></i>
-        Back to news list
-      </a>
+      <header>
+        <a class="btn btn-primary mb-4" href="<%= urlPrefix %>/news_items" role="button">
+          <i class="fa fa-arrow-left" aria-hidden="true"></i>
+          Back to news list
+        </a>
 
-      <h1>
-        <%= news_item.title %>
-      </h1>
+        <h1>
+          <%= news_item.title %>
+        </h1>
 
-      <p class="text-muted">
-        <i>
-          Posted on <%= news_item.formatted_date %>
-          <% if (news_item.author) { %>
-          by <%- news_item.author %>
+        <p>
+          <i class="mr-2 text-muted">
+            Posted on <%= news_item.formatted_date %>
+            <% if (news_item.author) { %>
+            by <%- news_item.author %>
+            <% } %>
+          </i>
+          <% if (news_item.show_student_badge) { %>
+              <span class="badge badge-secondary">Visible to students</span>
           <% } %>
-        </i>
-        <% if (news_item.show_student_badge) { %>
-            <span class="badge badge-secondary ml-2">Visible to students</span>
-        <% } %>
-      </p>
+        </p>
+      </header>
 
       <%- news_item_html %>
-
-    </div>
+    </article>
+    <aside class="container">
+      <hr class="mt-5">
+      <p class="text-right small">
```

Comment:
```suggestion
      <p class="text-right small border-top mt-5 pt-3">
```
Preferable to avoid styling html (`<hr>`, `&nbsp;`, empty `<p></p>`, etc). Alternatively, we can make the horizontal rule `aria-hidden`.